CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Underline/strikeout drawn above visual bell
 - Terminal going transparent during visual bell
 - Selection not being cleared when sending chars through a binding
+- Mouse protocol/encoding handling not following specification
```

Comment:
I had trouble finding the specification, I doubt others would have an easy time.

We should just describe what exactly we fixed so people don't have to go hunting for this mysterious 'specification'.

Something like this:
```suggestion
- Mouse protocols/encodings not being mutually exclusive within themselves
```